### PR TITLE
Message explaining missing Report Diff.

### PR DESCRIPTION
**Pull request by Parakleta**
As per #938

### CODE REVIEW

**Comment by Parakleta** [comment]
I've signed the CLA, but the 'check the status' link just returns 404.

**Comment by leobalter** [comment]
maybe it needs an empty commit to check the CLA again.

cc @jzaefferer

**Comment by trentmwillis** [comment]
Seems like maybe the email or full name you provided might not match up with your git config. You can check what those are set as by running `git config -l` in your repository. The `user.name` and `user.email` properties should match what you put in the CLA.

**Comment by Parakleta** [comment]
I don't use the cli tools for PRs, just the built in github editor.  If you click on the commit and add `.patch` to the end of the URL you get the patch email which has 'Parakleta <parakleta@darkreality.org>' as the `From: ` field which is the name and email I entered in the CLA.  How does your CLA tool work?

**Comment by trentmwillis** [comment]
I'm not super familiar with that code, but it looks like it may require the name to contain at least two parts: https://github.com/jquery/jquery-license/blob/master/lib/validate.js#L18-L25

@jzaefferer seems like he would have more insight.

**Comment by Parakleta** [comment]
That looks like it, someone will have to manually put it through then.

**Comment by jzaefferer** [comment]
I don't have access to do the manual confirmation. @scottgonzalez can you help out here?

**Comment by kborchers** [comment]
This should be fixed now @leobalter @Parakleta. Sorry for the delay.

**Comment by trentmwillis** [comment]
Thanks again @kborchers!

@Parakleta sorry for the wait on this, but would you mind adding a test case for this as well?

**Comment by leobalter** [comment]
for the records, @csnover was coming to help too. 

Thanks @kborchers and @csnover.

**Comment by leobalter** [comment]
LGTM

File: reporter/html.js [hunk 1]
```diff
@@ -713,6 +713,10 @@ QUnit.log( function( details ) {
 				" run with a higher max depth or <a href='" +
 				escapeText( setUrl( { maxDepth: -1 } ) ) + "'>" +
 				"Rerun</a> without max depth.</p></td></tr>";
+		} else {
+			message += "<tr class='test-message'><th>Message: </th><td>" +
```
Comment (leobalter) [inline reporter/html.js]: this is consistent with [line 710](https://github.com/jquery/qunit/pull/944/files#diff-5b7e94e2a10b848c4ae4b539388f86aeR710) but I wonder if we should keep this `<th>Message: </th>`.

Anyway, this should be addressed in another patch.

**Comment by leobalter** [comment]
Thanks, @Parakleta! This is merged at  8460ced
